sdk/test/action-handler-test.js

```diff
@@ -130,7 +145,7 @@ describe('ActionHandler', () => {
   });
 
   // synchronous handlers
-
+  /*
```

Comment:
Assuming all the other tests were commented out for faster testing, but should still be enabled.

```suggestion
```

Reply:
Ah, ups, thanks for re-enabling